translator/combine_wzdx.py

```diff
@@ -1,15 +1,22 @@
-import json
+import argparse
 import copy
-from shapely.geometry.polygon import Polygon
-from shapely.geometry import Point
-import geopy
-from geopy.distance import geodesic
-import pyproj
 import getopt
```

Comment:
Several imports seem to be unused here, can we clean up?

Reply:
removed unused imports (and sorted imports) across all translators, tools, and tests